fuse/dl/models/heads/common.py

```diff
@@ -46,6 +46,31 @@ def forward(self, x):
         x = self.classifier(x)
         return x
 
+class ClassifierFCN3D(nn.Module):
```

Comment:
can you give a short description? 
also to explain how it's different from Head3dClassifier